docs/pages/tutorials/github-oauth/sveltekit.md

```diff
@@ -100,11 +104,10 @@ Create an API route in `routes/login/github/+server.ts`. Generate a new state, c
 
 ```ts
 // routes/login/github/+server.ts
```

Comment:
Can you revert the imports. The project separates JS and type imports

Reply:
Oh ok, because I follow npm and alphabetical order.

Reply:
Done